Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/LightingUtils.azsli

```diff
@@ -11,13 +11,17 @@
 #include <Atom/RPI/Math.azsli>
 
 //! Transform cubemap coordinates to the expected cubemap orientation
+half3 GetCubemapCoords(half3 original)
+{
+    return half3(-original.x, original.z, -original.y);
```

Comment:
Should this function accept `real` types?

Reply:
Yes I could. This was part of my attempt to satisfy precision loss warnings coming from other android shaders that were not converted over to use real keyword. In the end I gave up on that effort and basically the plan is to just live with those warnings until we convert over all the other relevant shaders.